File: study-2nd/level1/jiwon/solution1.js
function solution(s){
    if(s.match(/p/gi).length == s.match(/y/gi).length){
        return true;
    }else if(s.match(/p/gi).length && s.match(/y/gi).length == 0){
        return true;
    }else{
        return false;
    }
}

File: study-2nd/level2/jihye/Solution1.java
class Solution {
    public String solution(String s) {
        // answer: Jaden Case로 바꾼 문자열
        String answer = "";

        for (int i = 0; i < s.length(); i++) {
            // 문자열의 맨 첫번째 문자이거나 바로 앞이 공백인 문자인 경우
            if (i == 0 || s.charAt(i-1) == ' ') {
                answer += Character.toString(s.charAt(i)).toUpperCase();
            } else {
                answer += Character.toString(s.charAt(i)).toLowerCase();
            }
        }
        return answer;
    }
}
